File: Group6System-main/Group6System-main/ApplicationTier/src/main/java/com/example/applicationtier/models/Address.java
package com.example.applicationtier.models;

import com.fasterxml.jackson.annotation.JsonProperty;
import org.springframework.stereotype.Component;

@Component
public class Address {
    @JsonProperty("city")
    private City city;

    @JsonProperty("streetname")
    private String streetName;

    @JsonProperty("streetnumber")
    private String streetNumber;

    public Address() {
    }

    public City getCity() {
        return city;
    }

    public void setCity(City city) {
        this.city = city;
    }

    public String getStreetName() {
        return streetName;
    }

    public void setStreetName(String streetName) {
        this.streetName = streetName;
    }

    public String getStreetNumber() {
        return streetNumber;
    }

    public void setStreetNumber(String streetNumber) {
        this.streetNumber = streetNumber;
    }
}

File: Group6System-main/Group6System-main/BlazorClient/BlazorClient/Models/Address.cs
using System;
using System.Text.Json.Serialization;

namespace BlazorClient.Models
{
    public class Address
    {
        [JsonPropertyName("addressid")]
        public int AddressId { get; set; }
        [JsonPropertyName("city")]
        public City City { get; set; }
        
        [JsonPropertyName("streetname")]
        public String StreetName { get; set; }
        
        [JsonPropertyName("streetnumber")]
        public String StreetNumber { get; set; }
    }
}